cli/commands/benchmark-web-vitals.mjs

```diff
@@ -161,13 +161,28 @@ export async function handler( opt ) {
 
 	const browser = await puppeteer.launch( { headless: 'new' } );
 
+	// Log progress only under certain conditions (multiple URLs to benchmark).
+	const logProgress = logURLProgress( opt );
+
 	for await ( const url of getURLs( opt ) ) {
-		const { completeRequests, metrics } = await benchmarkURL(
-			browser,
-			params
-		);
+		if ( logProgress ) {
+			log( `Benchmarking URL ${ url }...` );
+		}
 
-		results.push( [ url, completeRequests, metrics ] );
+		// Catch Puppeteer errors to prevent the process from getting stuck.
+		try {
+			const { completeRequests, metrics } = await benchmarkURL(
```

Comment:
What about progress for benchmarking an individual URL? If I have 100 requests for each URL, I'd love to be able to see progress for how far along it is when analyzing each URL.

Reply:
Makes sense to add this. I would suggest we only do so if more than 1 request is being made, as otherwise there wouldn't be any value.